templates/index

```diff
@@ -1,368 +1,369 @@
-# First line is the header fields for columns and is mandatory.
```

Comment:
Can you please revert whatever formatting was done on this and just make sure your new template line is the only change?

Reply:
sorry my bad